scripts/setup-env.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+set -ex
+
+CCF_VERSION=${CCF_VERSION:-"6.0.1"}
+CCF_PLATFORM=${PLATFORM:-"virtual"}
+
+tdnf update -y
+
+tdnf install -y \
+    git \
+    build-essential \
+    python3-pip \
+    ca-certificates \
+    jq \
+    which \
+    procps \
+    clang-tools-extra-devel
```

Comment:
please use specific versions to make it transparent what is installed and to make it reproducible

Reply:
IMO this is not as important as dockerfile